package.json

```diff
@@ -41,8 +41,8 @@
   },
   "homepage": "https://github.com/WickyNilliams/react-simple-colorpicker",
   "peerDependencies": {
-    "react": "^0.14.0",
-    "react-dom": "^0.14.0"
+    "react": "^0.14.0 || ^15.1.0",
+    "react-dom": "^0.14.0 || 15.1.0"
```

Comment:
I think you accidentally dropped a `^` on the react-dom 15.1 dependency.

I've also seen some projects use `"react": ">=0.14.0"` on their peer-dep